proxy.py

```diff
@@ -1481,7 +1479,10 @@ def load_plugins(plugins: bytes) -> Dict[bytes, List[type]]:
         if plugin == b'':
             continue
         module_name, klass_name = plugin.rsplit(DOT, 1)
-        module = importlib.import_module(text_(module_name))
+        if module_name == 'proxy':
+            module = importlib.import_module('__main__')
+        else:
+            module = importlib.import_module(text_(module_name))
```

Comment:
I'll also prefer something like:

```
module_name = '__main__` if module_name == b'proxy' else text_(module_name)
module = importlib.import_module(module_name)
```

Wdyt?

Reply:
I think the former is more clear although it have used more spaces.

Reply:
Sounds good.